math/mode/mode.go

```diff
@@ -0,0 +1,40 @@
+// mode_test.go
+// author(s): [CalvinNJK] (https://github.com/CalvinNJK)
+// description: Finding Mode Value In an Array
+// see mode.go
+
+package mode
+
+func Mode(numbers []float64) float64 {
+
+	// Find the size of the array
+	n := len(numbers)
+
+	// declare variables
+	maxCount := 0
+	maxValue := 0.0
+
+	// Loop for the whole array
+	for i := 0; i < n; i++ {
+
+		count := 0
+
+		// check the selected Value have any repeated same value
```

Comment:
These comments are of no use, everyone who can read the code can see what you're trying to accomplish. When commenting on code, describe "why" instead of "what".